### PR TITLE
TILA-1071 | Reservation Deny reason

### PR DESCRIPTION
**Pull request by nicobav**
Adds model for reason for denying the reservation like the `ReservationCancelReason`. This is used by staff users to fill in predefined reason for some reservations that requires handling.

TILA-1071

changes for handle mutations in separate PRs

### CODE REVIEW

File: reservations/tests/factories.py [hunk 1]
```diff
@@ -14,6 +14,11 @@ class Meta:
         model = "reservations.ReservationCancelReason"
 
 
+class ReservationDenyReasonFactory(DjangoModelFactory):
+    class Meta:
+        model = "reservations.ReservationCancelReason"
```
Comment (jmp) [inline reservations/tests/factories.py:19]: Should be `ReservationDenyReason`.

**Review by jmp — CHANGES_REQUESTED** [review]

File: reservations/tests/factories.py (hunk 1 above)
Comment (nicobav) [reply reservations/tests/factories.py]: Good catch. I actually fixed this one in the draft PR but it should be here :woozy_face:

**Comment by nicobav** [comment]
fixed those copypaste errors

**Review by jmp — APPROVED** [review]
Looks good now! 👍

**Pull request merged**